Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -248,19 +248,27 @@ extension KeyboardShortcuts {
 					return nil
 				}
 
-				guard !shortcut.isTakenBySystem else {
+				if shortcut.isTakenBySystem {
 					self.blur()
 
-					NSAlert.showModal(
+					let modalResponse = NSAlert.showModal(
 						for: self.window,
 						title: "keyboard_shortcut_used_by_system".localized,
 						// TODO: Add button to offer to open the relevant system settings pane for the user.
-						message: "keyboard_shortcuts_can_be_changed".localized
+						message: "keyboard_shortcuts_can_be_changed".localized,
+						buttonTitles: [
```

Comment:
The first item should be `OK`, the second item should be `Use Anyway`, since the default choice should be `OK`.

Reply:
Hi @sindresorhus, I'm a great fan of your work. Using many of the projects you released for the apps I'm working on – thanks.

I had a quick look at the Apple HIG on this, and this example on alerts is the closest I could get to. 
<img width="521" alt="Screenshot 2023-04-04 at 22 23 22" src="https://user-images.githubusercontent.com/73470/229807790-0c8af211-7fc0-4e2b-8827-c203cc693e6a.png">

So things boil down to what the primary choice would be in this situation.

My rationale for the order of the buttons, which I think fits with what's outlined in the HIG, is: a 'cancel' action at the left (or bottom now?) for the user to go back and re-take an action, or a 'go ahead' action at the right (or top?) for her to proceed with the choice, having digested the message in the dialog. I.e. 'go ahead' should be the primary action.

So I think that, even if the action were to be interpreted as strongly inadvisable, the button to go ahead should be the first one. Else, the default choice would be to cancel the action, which I can't remember to be the order of choices in alerts I've seen. 

I'm happy to be proven wrong with any example alert available in a recent version of macOS though!

Reply:
In this case, the primary button should be "OK." The primary button should represent the default, recommended, or most likely action that users would take. Since "Use Anyway" is not recommended, it should not be the primary button. Making "OK" the primary button will help guide users towards the safer and more desirable action. This follows user interface design best practices, which prioritize guiding users to make the right decisions and avoid potential issues.

This one is closer to ours:

![Screen Shot 2018-10-10 at 09](https://user-images.githubusercontent.com/170270/230138516-4be05be7-03e2-4b4c-aa11-5e9901dadb4a.png)